suite-native/navigation/src/components/TabBar.tsx

```diff
@@ -36,18 +39,20 @@ export const TabBar = ({
     tabItemOptions,
     SendReceiveComponent,
 }: TabBarProps) => {
-    const [sendReceiveActionsVisible, setSendReceiveActionsVisible] = useState(false);
+    const [isSendReceiveActionsVisible, setIsSendReceiveActionsVisible] = useAtom(
```

Comment:
I really love how simple and clean this is compared to redux solution, definitely good way to use jotai instead.

Reply:
Same. I'm down to use this for smaller things that just need to get propagated across the app. 

For example I probably wouldn't use this for accounts and transactions handling, but stuff like this and discreet mode which we discussed recently, it's just super nice to use this for it. 

There is really no need to create action, reducer case, selector and default value in state, if you can do all that in one line with this. 

Hooked 😅